release-helper.sh

```diff
@@ -1,18 +1,45 @@
 #!/usr/bin/env sh
 
+# Usage:
+### Push HEAD to staging and staging-mainnet
+# developer-onboarding/release-helper.sh push test
+#
+#
+### Push HEAD to all 3 stages
+# developer-onboarding/release-helper.sh push production
+#
+#
+### Push the current branch, instead of HEAD, to staging and staging-mainnet
+# developer-onboarding/release-helper.sh push branch
```

Comment:
hmm, the arguments on the `release-helper` script are little unintuitive because you'd expect `branch` and `test` to be orthogonal (because they're spaced like arguments), whereas they are mutually exclusive.

It seems that we have two arguments: `--source` which is the local branch that defaults to `HEAD` and `--destination` which takes one of `test` or `production`.

I don't think we need to refactor, but I recommend flattening the commands to make it clear they're mutually exclusive and the arguments are not composable:
`push-test`, `push-production`, and `push-branch`